src/common-components/Logistration.jsx

```diff
@@ -12,19 +16,46 @@ import { RegistrationPage } from '../register';
 
 const Logistration = (props) => {
   const { intl, selectedPage } = props;
+  const [institutionLogin, setInstitutionLogin] = useState(false);
+
+  const handleInstitutionLogin = () => {
+    sendTrackEvent('edx.bi.institution_login_form.toggled', { category: 'user-engagement' });
+    sendPageEvent(selectedPage === LOGIN_PAGE ? 'login' : 'register', 'institution_login');
```

Comment:
The category should remain the same `login_and_registration` for all three, only change the name according to the scenario. e.g. `institution_login`, `login`, `register`

Reply:
@waheedahmed I have updated the logic for the page event name. Please review again